examples/type-cast/tests/type_cast_test.bal

```diff
@@ -0,0 +1,44 @@
+import ballerina/test;
+import ballerina/io;
```

Comment:
Would it be better to reorder the imports alphabetically?

Reply:
Fixed!